DQM/Physics/src/TopSingleLeptonDQM.cc

```diff
@@ -380,11 +396,11 @@ void MonitorEnsemble::fill(const edm::Event& event,
 
   ------------------------------------------------------------
   */
-  if (!event.eventAuxiliary().run()) return;
-  fill("RunNumb_", event.eventAuxiliary().run());
+//  if (!event.eventAuxiliary().run()) return;
```

Comment:
hi @kuyoun - go ahead and remove these lines if they are no longer needed

Reply:
Removed